service/block/store.go

```diff
@@ -1,23 +0,0 @@
-package block
-
-import (
-	"context"
-
-	"github.com/celestiaorg/rsmt2d"
-
-	"github.com/tendermint/tendermint/pkg/da"
-
-	"github.com/celestiaorg/celestia-node/ipld"
-)
-
-func (s *Service) StoreBlockData(ctx context.Context, data *ExtendedBlockData) error {
-	shares := ipld.ExtractODSShares(data)
-	// TODO @renaynay: it's inefficient that we generate the EDS twice: once in event loop
-	// once in the IPLD plugin.
-	_, err := ipld.PutData(ctx, shares, s.store)
-	return err
-}
-
-func (s *Service) GetBlockData(ctx context.Context, dah *da.DataAvailabilityHeader) (*ExtendedBlockData, error) {
-	return ipld.RetrieveData(ctx, dah, s.store, rsmt2d.NewRSGF8Codec())
```

Comment:
Where does the equivalent of this happen now?

Reply:
ShareService

Reply:
for bridge nodes, [here](https://github.com/celestiaorg/celestia-node/blob/539ac21a965c5680d72daefa797a762b02e63325/service/header/header.go#L39).

for full nodes, [here](https://github.com/celestiaorg/celestia-node/blob/539ac21a965c5680d72daefa797a762b02e63325/service/share/full_availability.go#L29).